containerized_analytics/smile/docker-compose-smile.yml

```diff
@@ -9,16 +9,17 @@ services:
 
   minio:
     image: minio/minio
+    container_name: minio
     hostname: minio
     ports:
       - 9000:9000
       - 9001:9001
     volumes:
-      - "smile_content_data:/tmp"
-    command: server /tmp
+      - "smile_content_data:/data"
     environment:
-      - MINIO_ACCESS_KEY=${AWS_ACCESSKEY}
-      - MINIO_SECRET_KEY=${AWS_ACCESSKEYSECRET}
+      - MINIO_ROOT_USER=${AWS_ACCESSKEY}
```

Comment:
In docker-compose-smile-traefik.yml, it is like AWS_ACCESS_KEY and in here, it is MINIO_ROOT_USER. Can we make it as just one name that uses the ${AWS_ACCESSKEY} and ${AWS_ACCESSKEYSECRET}. It is quite confusing.

Reply:
Then maybe, we'd better change the one in the traefik as well. Maybe, let's set good names first, then update the variables names using them. Even if we need to connect to AWS, we can set the variable names to whatever we want, then use the values as AWS key values

Reply:
I created a task for that https://github.com/ncsa/standalone-smm-analytics/issues/96
Will address that when I have the bandwidth